iplover-app/www/js/pages/edit_view_nest_site.js

```diff
@@ -115,6 +115,11 @@ var save_form = function(e){
     
     //only if set to edited
     rec.changes_synced = false;
+	
+	//Time of last Edit. One is for server, one is for calculations
```

Comment:
indentation differences, looks like existing code is spaces, not tabs. Would switch to spaces.